core/templates/dev/head/domain/collection/CollectionNodeObjectFactory.js

```diff
@@ -17,20 +17,30 @@
  * collection node domain objects.
  */
 
-// TODO(bhenning): Implement validation functions and related tests.
+// TODO(bhenning): This should be provided by the backend.
```

Comment:
Discussed in chat with @BenHenning. Decision:

(1) add a comment similar to the one [here](https://github.com/oppia/oppia/blob/develop/core/templates/dev/head/domain/collection/CollectionUpdateService.js#L22), so that developers are aware of the coupling to the backend constant
(2) file an issue on the issue tracker explaining the steps needed for consolidating the constants, so that it can be taken up as a starter project